.gitignore

```diff
@@ -158,3 +158,5 @@ java/**/.settings
 java/**/.classpath
 java/**/.project
 
+# python virtual env
+venv
```

Comment:
I don't use python virtual env...

Reply:
yes... but it is a good habit to use virtual env.

it can be other directory names, I think `venv` is the most popular.